rules/0.15-block-typing.py

```diff
@@ -0,0 +1,12 @@
+import os
```

Comment:
Why does this rule come after the unanimity rule? Would we ever want to override the typing requirement?

Reply:
I think we should leave the unaminity rule in #1 so that if we screw things up we can rescue ourselves. While I don't see a way typing would break things I think it should be at most #2 out of precaution.